Software/Python/easygopigo.py

```diff
@@ -470,7 +470,23 @@ def read_raw_sensors(self):
         _wait_for_read()
 
         _grab_read()
+        
         five_vals = line_sensor.read_sensor()
+        five_vals=[0]*5
+        white_line=line_sensor.get_white_line()
+        black_line=line_sensor.get_black_line()
+        range_sensor= line_sensor.get_range()
+        threshold=[a+b/2 for a,b in zip(white_line,range_sensor)]
+        # print (white_line,black_line,range_sensor,threshold)
+        raw_vals=line_sensor.read_sensor()
```

Comment:
This line is the only one that should be between the _wait_for_read/_grab_read and the _release_read()